integration/k8s_fixture_test.go

```diff
@@ -161,18 +160,6 @@ func (f *k8sFixture) ClearResource(name string) {
 	}
 }
 
-func (f *k8sFixture) CreateNamespaceIfNecessary() {
```

Comment:
this seems cheap, is it worth leaving in?

```
$ time kubectl apply -f ns.yaml
namespace/mynamespace unchanged

real	0m0.145s
user	0m0.120s
sys	0m0.037s
```

Reply:
...okay hmm I guess it's not thaaat cheap when it's a no-op... do you know when/why it was originally put in?